README.md

```diff
@@ -1,23 +1,50 @@
-# Jobs Design Tokens
+<div align="center">
+<img src="https://github.com/lmc-eu/jobs-design-system/blob/main/static/jobs.svg?raw=true" width="422" height="300" alt="Jobs Design System" />
```

Comment:
I just noticed, that this doesn't exist, right?

Reply:
Removed.